docs/enterprise.md

```diff
@@ -0,0 +1,104 @@
+# Logfire Enterprise
+
+## Overview
+
+In addition to the [Pro plan](https://pydantic.dev/pricing), Logfire has two enterprise offerings:
+
+- **Enterprise Cloud**: Fully managed, SLA-backed service
+- **Enterprise Self-Hosted**: On-premises deployment via Kubernetes
+
+> **Note:** The Helm chart for Pydantic Logfire is now [open source](https://github.com/pydantic/logfire-helm-chart)
+
+## Enterprise Cloud
+
+### Target Users
+
+Organizations requiring:
+
+- Service Level Agreements (SLAs)
+- Custom Data Processing Agreements (DPAs)
+- HIPAA Business Associate Agreements (BAAs)
+- Invoice Billing
+- Custom retention
+
+### Features
+
+| Feature | Description |
+|---------|-------------|
+| **Full-Service Hosting** | All operational complexity managed by our engineering team |
+| **Enhanced Support** | Priority support with guaranteed SLAs, 24/7 dedicated assistance |
+| **Compliance & Custom Billing** | Tailored billing options and industry-specific compliance packages |
+| **Custom Retention** | Extended data retention beyond the standard 30-day SaaS offering |
+
+## Enterprise Self-Hosted
+
+### Target Users
+
+Organizations with:
+
+- Highly sensitive data requirements
+- Data sovereignty requirements
+- Kubernetes expertise
+
+### Features
+
+| Feature | Description |
+|---------|-------------|
+| **Deployment** | Open-sourced Helm chart for quick deployment on any Kubernetes cluster |
+| **Storage & Retention** | Parquet storage on any S3-compatible object storage with customizable retention policies |

```

Comment:
"Parquet storage on any S3-compatible object storage" sounds more like a requirement, what's the benefit being communicated here?

Reply:
any S3-compatible object storage - i.e. easily available